packages/hawtio/src/plugins/shared/jolokia-service.ts

```diff
@@ -312,14 +312,15 @@ class JolokiaService implements IJolokiaService {
       switch (xhr.status) {
         case 401:
         case 403: {
-          const url = new URL(window.location.href)
+          const url = new URL(decodeURIComponent(window.location.href))
```

Comment:
why do we need this?  Isn't it enough to encode the value assigned to `window.location.href` below?

Reply:
my understanding is that URL constructor isn't trying to decode the URL so if the URL is encoded this intentionally can throw an exception. If The URL isn't encoded  decode function returns same string. So there is no harm.

Reply:
> " if the URL is encoded this intentionally can throw an exception."

The only encoded part should be query if there are any, so why do we need to prepare for the case?   Is there a valid case where the entire URL may be encoded?

Reply:
you're right probably not. At least I haven't found any and I just wanted to be sure.  So I will remove it.